server/main.py

```diff
@@ -1,7 +1,101 @@
-from fastapi import FastAPI
+from fastapi import Depends, FastAPI, HTTPException, status
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+# from model import User
+import model
+from database import SessionLocal, engine
+from typing import Union
+
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+from datetime import datetime, timedelta
+
+model.Base.metadata.create_all(bind=engine)
+
+# Acquired by CLI "openssl rand -hex 32"
+SECRET_KEY = "9ff8f5a166924e4cc01ad9682302abf54ac778cf794cbf642074bf3ed58ccaf0"
```

Comment:
This should be secret
ref to https://github.com/tymondesigns/jwt-auth/issues/311

Reply:
@sungminoh SECRET_KEY to .env file?

Reply:
yes.